core/src/main/java/com/hcl/ecomm/core/services/impl/CreateCartServiceImpl.java

```diff
@@ -47,16 +47,23 @@ public String getEmptyCartPath() {
 	
 
 	@Override
-	public JSONObject createGuestCart() {
+	public JSONObject createCart(String customerToken) {
 		LOG.debug("createGuestCart method start.");
 		String scheme = "http";
+		String authToken = "";
+		String createGuestCartPath = "";
```

Comment:
@DeepaliAgrwl : replace createGuestCart to createCart everywhere required

Reply:
Done